### PR TITLE
github: add bug report template

### PR DESCRIPTION
**Pull request by mfkl**
Suggestions welcome :-)

### CODE REVIEW

File: .github/workflows/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG] Summary of item"
+labels: bug
```
Comment (jgiannuzzi) [inline .github/workflows/ISSUE_TEMPLATE/bug_report.md:5]: I think we don't need `[BUG]` in the title as the `bug` label is set

File: .github/workflows/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+assignees: ''
+
+---
+
+<!--
+Thanks for reporting this! Please provide as much info as you can about the issue, so we can best help you. As a general guidance, pointers from https://stackoverflow.com/help/mcve can help you craft a good bug report.
+
+If this is a question please ask on [StackOverflow](https://stackoverflow.com/questions/tagged/consul).
```
Comment (jgiannuzzi) [inline .github/workflows/ISSUE_TEMPLATE/bug_report.md:13]: Maybe we should say "If this is a question _about the Consul API or Consul itself_"?

File: .github/workflows/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+
+
+**Steps To Reproduce**
+<!--
+Provide the steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
```
Comment (jgiannuzzi) [inline .github/workflows/ISSUE_TEMPLATE/bug_report.md:27]: As this is a library, I would maybe expect that the steps to reproduce would likely be a piece of code and maybe the Consul config and/or command used to launch the agent. Maybe we should rephrase those steps to reflect that?

File: .github/workflows/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+
+
+**Expected behavior**
+<!-- A clear and concise description of what you expected to happen. -->
+
+
+
+**Screenshots** or **Logs**
+<!-- If applicable, add screenshots to help explain your problem. -->
```
Comment (jgiannuzzi) [inline .github/workflows/ISSUE_TEMPLATE/bug_report.md:38]: I think we should emphasize logs over screenshots here, or even remove screenshots altogether.
If user post screenshots of logs/code/config, they won't be very accessible.

File: .github/workflows/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+
+
+**Environment**
+<!-- Please complete the following information. -->
+- OS:       <!-- [e.g. Windows] -->
+- Consul Version   <!-- [e.g. 1.6.1.1] -->
```
Comment (jgiannuzzi) [inline .github/workflows/ISSUE_TEMPLATE/bug_report.md:45]: We should clarify that we want both the version of Consul, and the version of this library.

File: .github/workflows/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (mfkl) [reply .github/workflows/ISSUE_TEMPLATE/bug_report.md]: removed

File: .github/workflows/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (mfkl) [reply .github/workflows/ISSUE_TEMPLATE/bug_report.md]: done

File: .github/workflows/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (mfkl) [reply .github/workflows/ISSUE_TEMPLATE/bug_report.md]: done

File: .github/workflows/ISSUE_TEMPLATE/bug_report.md (hunk 4 above)
Comment (mfkl) [reply .github/workflows/ISSUE_TEMPLATE/bug_report.md]: removed

File: .github/workflows/ISSUE_TEMPLATE/bug_report.md (hunk 5 above)
Comment (mfkl) [reply .github/workflows/ISSUE_TEMPLATE/bug_report.md]: added

**Comment by jgiannuzzi** [comment]
Hey @mfkl, could you please move the issue template into `.github/ISSUE_TEMPLATE` instead of `.github/workflows/ISSUE_TEMPLATE`?

I should have seen that before merging 🤷

**Pull request merged**